src/validation/custom.validation.ts

```diff
@@ -98,16 +98,18 @@ export const checkOptionalDate = (dayStr: string = "", monthStr: string = "", ye
 
 export const checkIdentityDate = (dayStr: string = "", monthStr: string = "", yearStr: string = "") => {
   // to prevent more than 1 error reported on the date fields we check if the year is correct length or missing before doing the date check as a whole.
```

Comment:
the comment will need updating as it says the length check is done first

Reply:
Updated comment